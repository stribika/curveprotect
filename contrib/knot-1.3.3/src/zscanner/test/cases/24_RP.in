$ORIGIN	.
$TTL	1

; The RP is the same as the MINFO, so there are the differences and basics only.

; OK
@	RP	mail	mail.tld.	; Relative and absolute dnames
@	RP	\# 2 00 00		; Hexadecimal rdata
@	TYPE17	\# 2 00 00		; TYPE + Hexadecimal rdata
@	TYPE17	@	@		; TYPE
@	rp	@	@		; Type in lower-case

; KO
@	RP
