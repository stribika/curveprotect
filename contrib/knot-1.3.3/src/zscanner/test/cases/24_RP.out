OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0011
RDATA=046D61696C00 046D61696C03746C6400
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0011
RDATA=00 00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0011
RDATA=00 00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0011
RDATA=00 00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0011
RDATA=00 00
------
WARNG=ZSCANNER_EBAD_RDATA
------
